src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -780,8 +814,12 @@ object ErgoWalletActor {
     * @param initialized   - whether wallet is initialized or not
     * @param unlocked      - whether wallet is unlocked or not
     * @param changeAddress - address used for change (optional)
+    * @param height        - last height scanned
     */
-  case class WalletStatus(initialized: Boolean, unlocked: Boolean, changeAddress: Option[P2PKAddress])
+  case class WalletStatus(initialized: Boolean,
+                          unlocked: Boolean,
+                          changeAddress: Option[P2PKAddress],
+                          height: ErgoHistory.Height)
```

Comment:
Why not name the field `lastScannedHeight`?

Reply:
I think this is obvious for an API client that height means current height of the wallet, i.e. height of the last block scanned.

Reply:
Not obvious at all since we have more than one kind of height. Better be explicit and unambiguous especially for API.

Reply:
In the wallet, there's one notion of height I guess. 
Please also note that in "lastScannedHeight" two first words are useless by the blockchain construction. The wallet's height is indeed last one, and that block is indeed scanned.

Reply:
Three word better describe the meaning than one. Why you don't want to be explicit, you are not sure about real meaning, or think it may change in future?

Reply:
I think my comment above explains my logic pretty well.